src/node/utils/LibreOffice.js

```diff
@@ -35,7 +35,18 @@ var queue = async.queue(doConvertTask, 1);
  * @param  {Function}   callback    Standard callback function
  */
 exports.convertFile = function(srcFile, destFile, type, callback) {
-  queue.push({"srcFile": srcFile, "destFile": destFile, "type": type, "callback": callback});
+  if (type === 'doc') {
+    queue.push({
+      "srcFile": srcFile,
+      "destFile": destFile.replace(/\.doc/, '.odt'),
```

Comment:
I don't know if `srcFile` could be something like `"test.html.html"`, but in this case this `replace` would not work as you're expecting. You can use the `$` regex mark of EOL to fix that:
```
"test.html.html".replace(/\.html/, '.odt')
> "test.odt.html"
"test.html.html".replace(/\.html$/, '.odt')
> "test.html.odt"
```